### ISSUE TITLE
Adjust heading levels on releases page

### ISSUE THREAD

**Issue by mxstbr** [styled-components/styled-components-website#202]
The `/releases` page on the website is auto-generated from the GitHub releases. (thanks to @SaraVieira) This can lead to situations where the headings inside a release are at the same level as the release itself:

<img width="1081" alt="screen shot 2017-12-30 at 21 18 55" src="https://user-images.githubusercontent.com/7525670/34457097-0e844270-eda7-11e7-97d3-442c54a86067.png">

Or even above it:

<img width="1060" alt="screen shot 2017-12-30 at 21 19 43" src="https://user-images.githubusercontent.com/7525670/34457099-2c8bb8f2-eda7-11e7-8c30-fefedb8bc0b2.png">

which is suboptimal. :confused:

The way to solve this would be to treat the incoming headings from the releases as relative. The version heading is always an `h2`, so the headings in the text should be treated as a level below that. So, let's say the text we get looks like this as the release content:

```
# Some heading

## Some smaller heading
```

Then the output should be

```html
<h3>Some heading</h3>

<h4>Some smaller heading</h4>
```

Note how it's level 1 + 2 headings in the incoming text, but we treat them as _relative_ to the base heading level of any release, which is `h2`. So those two headings should be rendered as an `h3` and an `h4`.

Now let's say the release actually has an second level heading first, and no first-level heading: (most of our release notes look like this)

```
## Some heading

### Some smaller heading
```

Then the output should be the exact same:

```html
<h3>Some heading</h3>

<h4>Some smaller heading</h4>
```

Even though there was a second- and a third-level heading in the content, since there's no `h1` in the entire content, treat the top-most level (in this case the second level) as the top-most level relative to our base level.

I don't know if I'm making any sense but essentially I just want to make sure that no matter who writes the release notes the structure of the `/releases` page on GitHub stays consistent. :blush:

**Comment by morajabi** [comment]
I personally think most of it is not worth the effort or at least the extra logic to maintain. It can be fine with a simple effort when writing the notes.

But I agree we can just change spacing, font size and the font weight, but I don't like manipulating the MD. What do you think?

**Comment by mxstbr** [comment]
Let's start with the easy bits first _for sure_, don't worry about tackling this.

**Comment by philpl** [comment]
I think it should be quite trivial to add an offset to the heading level :)

**Comment by halvves** [comment]
Closed by #215
